cypress/e2e/yves/order-management/cart-custom-order-reference.cy.ts

```diff
@@ -0,0 +1,33 @@
+import { container } from '@utils';
+import { CartCustomOrderReferenceStaticFixtures, CartCustomOrderReferenceDynamicFixtures } from '@interfaces/yves';
+import { MultiCartPage, CustomOrderReferenceCartPage } from '@pages/yves';
+import { CustomerLoginScenario } from '@scenarios/yves';
+
+describe('cart item quantity', { tags: ['@order-management'] }, (): void => {
```

Comment:
`custom-order-reference-management.cy.ts`

please move this file to backoffice folder, you should also ensure that the reference persists inside the order.